.github/workflows/ci.yml

```
@@ -38,7 +38,7 @@ jobs:
       fail-fast: false
       matrix:
         project:
-          ["intro/blinky", "intro/button", "intro/button-interrupt", "intro/hello-world", "intro/panic", "intro/http-client"]
+          ["intro/blinky", "intro/button", "intro/button-interrupt", "intro/hello-world", "intro/panic", "intro/http-client", "advanced/stack_overflow_detection]
```

```suggestion
          ["intro/blinky", "intro/button", "intro/button-interrupt", "intro/hello-world", "intro/panic", "intro/http-client", "advanced/stack_overflow_detection"]
```

Just noticed that CI was not triggered and its because of this typo. https://github.com/esp-rs/no_std-training/actions/runs/6098840580/workflow

---

Wait, it's already fixed!